src/main/java/driver/Driver.java

```diff
@@ -0,0 +1,83 @@
+package driver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import lombok.SneakyThrows;
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.URL;
+import java.time.Duration;
+
+import static util.Constants.GRID_URL;
+import static util.Constants.SAUCE_URL;
+
+public class Driver {
+
+    private static final String BROWSER = StringUtils.defaultString(System.getProperty("browser"), "chrome");
+    private static final String REMOTE_URL = StringUtils.defaultString(System.getProperty("remote"), "local");
+
+    private static Driver instanceOfDriver = null;
+    private WebDriver driver;
+
+    private Driver() {
+        if (REMOTE_URL.equals("local")) {
+            setDriver();
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(15));
+            driver.manage().window().maximize();
+        } else {
+            driver = getRemote(REMOTE_URL, getCapabilities());
+        }
+    }
+
+    private WebDriver setDriver() {
```

Comment:
make it `void` and remove `return driver;`

Reply:
Fixed